packages/wallets/src/components/Transition/Transition.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, PropsWithChildren, useEffect, useState } from 'react';
+
+type TTransitionProps = {
+    className?: string;
+};
+
+const Transition: FC<PropsWithChildren<TTransitionProps>> = ({ children, className }) => {
+    const [isMounted, setIsMounted] = useState(false);
+
+    useEffect(() => {
+        setIsMounted(true);
+    }, []);
```

Comment:
```suggestion
    const [isMounted, setIsMounted] = useState(false);

    useEffect(() => {
        setIsMounted(true);
    }, []);
```
Please replace this with useIsMounted https://usehooks-ts.com/react-hook/use-is-mounted from `usehooks-ts`

Reply:
Changed the implementation, now we don't need this component.